libs/components/forms/src/lib/modules/input-box/input-box.component.spec.ts

```diff
@@ -585,16 +594,22 @@ describe('Input box component', () => {
       fixture.detectChanges();
 
       const inputBoxEl = getInputBoxEl(fixture, 'input-inline-help');
-      const inputBoxFormControlEl = inputBoxEl.querySelector(
+      const inputBoxFormControlEl = inputBoxEl?.querySelector(
         '.sky-input-box-group-form-control'
       );
-      const helpBtn = inputBoxEl.querySelector('button');
+
+      if (!inputBoxFormControlEl) {
```

Comment:
Instead of checking if the input element exists, you can cast the variable to `HTMLElement`:
```
const inputBoxFormControlEl = inputBoxEl?.querySelector(
  '.sky-input-box-group-form-control'
) as HTMLElement;
```

Reply:
But `querySelector` can return `null` and so that would be a false typing. This if statement is only needed because `fireDomEvent` does not handle `undefined` (see my comments on checkbox).

Reply:
Fixed via the change we made in checkbox.